Creates incorrect injections for class without class name

__Describe the Bug__

Hi,
I noticed that minified output sometimes inlines classes I pass to didi.
In this case, it generates an unexpected dependency called `class`.

__Steps to Reproduce__

<!-- Clear steps that allow us to reproduce the error. -->

1. parse annotations with a class without name.
```js
        expect(parseAnnotations(class{})).to.eql([ ]);
```
2. expect no dependencies
3. parseAnnotations in this example returns `[ "class" ]`

```
      AssertionError: expected [ 'class' ] to deeply equal []
      + expected - actual

      -[
      -  "class"
      -]
      +[]
```


__Expected Behavior__

<!-- A clear and concise description of what you expected to happen. -->
Parsing this class should return an empty array or an array without `class`

__Environment__

- Host (Browser/Node version), if applicable: Node 16.17.0
- OS: linux
- Library version: 8.0.1

__Possible solution__

It seems like this happens because didi determines that a class is a class by checking if it [starts with `class `](https://github.com/nikku/didi/blob/master/lib/util.js#L1).
In the inlined case, the class can start with `class` without an extra space

Would it be ok to change the heuristic to check for `class ` or `class{`? I can create a PR with a test if that's ok.
I.e. a solution like the following would solve this issue:

```js
var CLASS_PATTERN = /^class[ {]/;` 
```

I've created https://github.com/nikku/didi/pull/18 which implements this solution

--- a/lib/util.js
+++ b/lib/util.js
@@ -1,4 +1,4 @@
-var CLASS_PATTERN = /^class /;
+var CLASS_PATTERN = /^class[ {]/;
 
 
 /**

--- a/test/annotation.spec.js
+++ b/test/annotation.spec.js
@@ -156,6 +156,14 @@ describe('annotation', function() {
         expect(parseAnnotations(Car)).to.eql([ ]);
       });
 
+
+      it('without class name', function() {
+
+        // Disable keyword-spacing to reproduce #17
+        // eslint-disable-next-line keyword-spacing
+        expect(parseAnnotations(class{})).to.eql([ ]);
+      });
+
     });
 
 

